src/sagemaker/serve/schema/task.json

```diff
@@ -1,28 +1,28 @@
 {
 	"fill-mask": {
-        "sample_inputs": {
+		"sample_inputs": {
 			"properties": {
-				"inputs": "Paris is the <mask> of France.",
+				"inputs": "Paris is the [MASK] of France.",
```

Comment:
Thanks for fixing, you will need to fix corresponding UT as well [tests/unit/sagemaker/serve/utils/test_task.py](https://github.com/aws/sagemaker-python-sdk/pull/4429/files#diff-9c7523e4045175a02cb5a1fb56b029be37274d93a3036d7dc9452c31f04fbdf5)